telco-examples/mgmt-cluster/aarch64/README.md

```diff
@@ -0,0 +1,102 @@
+## Tech-Preview AARCH64 Integration (full e2e, mgmt-cluster and downstream clusters using aarch64 architecture)
+
+This is an example to demonstrate how to deploy a management cluster for Telco using SUSE ATIP and the fully automated directed network provisioning using aarch64 architecture.
+
+# Management Cluster in a single-node setup (aarch64)
+
+This is an example of using Edge Image Builder (EIB) to generate a management cluster iso image for SUSE ATIP. The management cluster will contain the following components:
```

Comment:
I'd suggest to specify the exact version of SUSE ATIP (e.g. `3.1.1`) and use its respective versions. Currently, the `mgmt-cluster-singlenode.yaml` definition mixes 3.1.0 versions with upstream.

Reply:
This is a tech-preview example, and we need to use the upstream repos because it will be release as a product feature in 3.3.